pkg/ui/ccl/src/views/clusterviz/containers/map/cpuSparkline.tsx

```diff
@@ -13,7 +14,7 @@ export function CpuSparkline(props: CpuSparklineProps) {
 
   return (
     <MetricsDataProvider id={key}>
-      <SparklineMetricsDataComponent>
+      <SparklineMetricsDataComponent formatCurrentValue={d3.format(".1%")}>
         <Metric name="cr.node.sys.cpu.sys.percent" sources={props.nodes} nonNegativeRate />
```

Comment:
reminder to remove the `nonNegativeRate`

Reply:
Done in #22747